paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentOption.kt

```diff
@@ -68,6 +77,21 @@ constructor(
         this.imageLoader = imageLoader
     }
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
I think a lot of these can be `internal` instead.

Reply:
I updated this and loadPaymentOption to be internal instead